R/splat-simulate.R

```diff
@@ -390,12 +390,12 @@ splatSimBatchCellMeans <- function(sim, params) {
     if (nBatches > 1) {
         batches <- colData(sim)$Batch
         batch.names <- unique(batches)
-
-        batch.facs.gene <- rowData(sim)[, paste0("BatchFac", batch.names)]
+        
+        batch.facs.gene <- as.data.frame(rowData(sim)[, paste0("BatchFac", batch.names)])
```

Comment:
Was there a reason for converting to `data.frame`? It's fine just curious in case it comes up somewhere else.

Reply:
Not converting it resulted in an incorrect dimensions error. I did update it again to just convert it to a matrix, the data.frame was not necessary.

Reply:
Hmmmm...ok. Wonder why it worked before then (at least I assume it did).